api/mesh/v1alpha1/dataplane_helpers_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 var _ = Describe("MultiValueTagSet", func() {
-
+	EnableInboundPassthrough = true
```

Comment:
I'd rather see this in BeforeSuit / BeforeAll / It section. Right now, the scope of this change is not really clear to me.